tests/config/commonFixtures.ts

```diff
@@ -151,9 +151,17 @@ export class TestChildProcess {
     this.exitCode = this.exited.then(r => r.exitCode);
   }
 
-  outputLines(): string[] {
+  outputLines(options: { prefix?: string } = {}): string[] {
     const strippedOutput = stripAnsi(this.output);
-    return strippedOutput.split('\n').filter(line => line.startsWith('%%')).map(line => line.substring(2).trim());
+    return strippedOutput
+        .split('\n')
+        .map(line => {
+          if (options.prefix && line.startsWith(options.prefix))
+            return line.substring(options.prefix.length);
+          return line;
+        })
+        .filter(line => line.startsWith('%%'))
```

Comment:
Hmm... I'd think `prefix` would replace `%%` in this function, but turns out it does not. I am worried this version will be hard to use in the future.

Reply:
agree. i've replaced this with a test-local implementation, `outputLines()` isn't even available on the result of `runInlineTest`.